docs/json/radarr/cf/truehd.json

```diff
@@ -61,6 +61,15 @@
       "fields": {
         "value": "\\bDD[^a-z+]|(?<!e)ac3"
       }
+    },
+    {
+      "name": "Not RlsGrp (TrueHD only)",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": true,
+      "required": true,
+      "fields": {
+        "value": "CtrlHD|W4NK3R|-DON"
```

Comment:
```suggestion
        "value": "\\b(CtrlHD|W4NK3R|-DON)\\b"
```

Reply:
`\b` means the `-` in `-DON` doesn't match for some reason. Could add the brackets for a capture group though.